test/unit/components/uploaderbutton.test.js

```diff
@@ -0,0 +1,48 @@
+/* global chai */
+/* global describe */
+/* global sinon */
+/* global it */
+/* global beforeEach */
+
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+
+import UploaderButton from '../../../app/components/uploaderbutton';
+
+const expect = chai.expect;
+
+describe('UploaderButton', function () {
+  const props = {
+    buttonUrl: 'http://tidepool.org/products/tidepool-uploader/',
+    buttonText: 'Get the Tidepool Uploader',
+    onClick: sinon.spy()
+  };
+
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
+      <UploaderButton
+        {...props}
+      />
+    );
+  });
+
+  it('should be a function', function() {
+    expect(UploaderButton).to.be.a('function');
+  });
+
+  describe('render', function() {
+    it('should render without problems', function () {
+      expect(wrapper.find(UploaderButton)).to.have.length(0);
+    });
+
+    it('should have provided button text', function () {
+      expect(wrapper.find('.btn-uploader').someWhere(n => (n.text().search(props.buttonText) !== -1))).to.be.true;
+    });
+
+    it('should respond to an onClick event', () => {
```

Comment:
I'm not picky about the technique, although I think what we do elsewhere is just check for 1 (but we also include a spy.reset() in an `afterEach` method when there are multiple tests on spies).